web/ui/src/lib/clustersMap/campus/angouleme.ts

```diff
@@ -0,0 +1,63 @@
+import { Campus } from '../campus';
+import { Cluster } from '../cluster';
+import { ICampus } from '../types';
+
+//
+export class Angouleme extends Campus implements ICampus {
+  emoji = (): string => '🇫🇷';
+
+  name = (): string => 'Angouleme';
+
+  extractorRegexp = (): RegExp =>
+    /(?<clusterWithLetter>c(?<cluster>\d+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>p(?<workspace>\d+))/i;
+
+  clusters(): Cluster[] {
+    return [
+      new Cluster({
+        identifier: 'c1',
```

Comment:
The identifier cannot be `c1` has your location haven't `c` in location, on your campus your identifier start with the number of cluster directly

```suggestion
        identifier: '1',
```